model/cicd/registry.yaml

```
@@ -139,6 +173,54 @@ groups:
         brief: >
           The result of a pipeline run.
         examples: ["success", "failure", "timeout", "skipped"]
+      - id: cicd.pipeline.action
```

```suggestion
      - id: cicd.pipeline.action.name
```

we usually end up needing to define more properties, so we recommend 

https://github.com/open-telemetry/semantic-conventions/blob/1000be4d897fafbc8554b373267992e47c1b86ac/docs/general/naming.md?plain=1#L76-L78

---

I did the rename in a9f2be76b22553df3e633e404d676bcbc4c288a5.